src/Security/Authentication/Cookies/src/CookieAuthenticationHandler.cs

```diff
@@ -122,7 +123,7 @@ private AuthenticationTicket CloneTicket(AuthenticationTicket ticket, ClaimsPrin
 
         private async Task<AuthenticateResult> ReadCookieTicket()
         {
-            var cookie = Options.CookieManager.GetRequestCookie(Context, Options.Cookie.Name);
+            var cookie = Options.CookieManager.GetRequestCookie(Context, Options.Cookie.Name!);
```

Comment:
What happens if Options.Cookie.Name is null?

Reply:
In the default case, this is guaranteed to be non-null. The option configures it to a value and you cannot assign a null value to it.